### DISCUSSION TITLE
How to customize Legend template and RTL support in UWP charting?

### DISCUSSION THREAD

**Discussion by drivingtheproject** [Live-Charts/LiveCharts2#263] [General]
Hello ,

Thanks a lot for wonderful library. 
How can i customize the legend template for UWP , i can see that sample part is commented .
How I can support the RTL support? if I set the FlowDirection = RightToLeft , whole chart is getting mirrored ?

Thanks in advance.

Cheers,
Saurabh

**Comment by beto-rodriguez** [comment]
Thanks, yes the web site is missing that, it should be included in the next version of the docs.

But the sample is at the samples folder:

https://github.com/beto-rodriguez/LiveCharts2/blob/master/samples/UnoSample/UnoSample.Shared/LiveChartsSamples/General/TemplatedLegends/View.xaml
